contacts/contacts.service.js

```diff
@@ -0,0 +1,71 @@
+import {
+  data
+} from './data';
+import {
+} from './entities';
+import {
+  CONTACTS_SERVICE_OBJECT_NAME,
+} from './constants';
+import {
+  createPersonInstance,
+  localStorageService
+} from './helper';
+
+
+export class ContactsService {
+
+  constructor() {
+    this.internalArrayContacts = [];
+    this.init();
+  }
+
+  init = () => {
+    const localStorageObject = localStorageService.getFromLocalStorage(CONTACTS_SERVICE_OBJECT_NAME);
+    this.internalArrayContacts = (localStorageObject && localStorageObject.length) ? localStorageObject.map(item =>
+      createPersonInstance(item)) : data.map(item => createPersonInstance(item)); // eslint-disable-line
+  }
+
+  getPersonById = personId => this.internalArrayContacts.find(item => item.id === personId);
+
+  addPerson = person => this.internalArrayContacts.unshift(person);
+
+  removePerson = (personId) => {
+    this.internalArrayContacts.splice(this.internalArrayContacts.findIndex(x => x.id === personId), 1);
```

Comment:
It's ok to  optimize your code, but you don't always need to over do it. In this situation you can add the index calculation  to a new variable to make the code a bit more readable.